File: docs/tutorial/cpp/src/main_initial.cpp
// Copyright © SixtyFPS GmbH <info@slint.dev>
// SPDX-License-Identifier: MIT

// ANCHOR: main
// src/main.cpp

#include "appwindow.h" // generated header from memory.slint

int main(int argc, char **argv)
{
    auto main_window = MainWindow::create();
    main_window->run();
}
// ANCHOR_END: main

File: docs/archive/reference/src/quickstart/main_initial.rs
// Copyright © SixtyFPS GmbH <info@slint.dev>
// SPDX-License-Identifier: MIT

#[allow(dead_code)]
// ANCHOR: main
fn main() {
    MainWindow::new().unwrap().run().unwrap();
}

slint::slint! {
    export component MainWindow inherits Window {
        Text {
            text: "hello world";
            color: green;
        }
    }
}
// ANCHOR_END: main
